pkg/agent/agent.go

```diff
@@ -105,7 +105,7 @@ func agentToken(ctx context.Context, agentNamespace, controllerNamespace string,
 		&v1.Namespace{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: agentNamespace,
-				Annotations: map[string]string{
+				Labels: map[string]string{
```

Comment:
We discussed and it's probably safer to remove this as we don't want to delete the agent namespace ("cattle-fleet-local-system") if has no clusters.